diff -uNr pianobooster-src-0.6.4b.old/src/CMakeLists.txt pianobooster-src-0.6.4b/src/CMakeLists.txt
--- pianobooster-src-0.6.4b.old/src/CMakeLists.txt	2009-11-27 22:25:35.000000000 +0100
+++ pianobooster-src-0.6.4b/src/CMakeLists.txt	2010-05-06 05:47:22.000000000 +0200
@@ -57,6 +57,8 @@
     #IF(ALSA_FOUND)
         ADD_DEFINITIONS(-D__LINUX_ALSASEQ__)
         LINK_LIBRARIES (asound)
+    LINK_LIBRARIES (pthread)
+    LINK_LIBRARIES (GL)
     #ELSE(ALSA_FOUND)
     #    MESSAGE(FATAL_ERROR "Please install the 'libasound2-dev' package and then try again")
     #ENDIF(ALSA_FOUND)
